### PR TITLE
Logout function

### PR DESCRIPTION
**Pull request by DelaMarc**
Added a logout function that clears the user cookies and redirects the user to the login page.
Also made a navigation bar template and included it on the according website pages. Thus, if the website's navbar has to be modified, we'll have only one file to change.

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -103,173 +114,160 @@ def Login():
                 error = 'Invalid password'
     return render_template('login.html', error = error)
 
-@app.route('/profile', methods = ['GET','POST'])
+@app.route('/logout', methods = ['GET', 'POST'])
```
Comment (YashasG98) [inline server.py]: Why exactly is GET required here?

File: templates/index.html [hunk 2]
```diff
@@ -39,7 +39,7 @@
             <span class="navbar-text">
             </span>
         </div>
-    </nav>
+    </nav>-->
     <div style='height: 100vh;' class="jumbotron">
         <h1 class="display-4">Hello, gamer!</h1>
         <p class="lead">Welcome to the abode of multiplayer gaming!</p>
```
Comment (YashasG98) [inline templates/index.html:18]: Please add the logout functionality to the exit button on this page. Also, while you're at it, rename it to 'Log out'

**Review by YashasG98 — CHANGES_REQUESTED** [review]
This is amazing @DelaMarc ! Please make the minor changes requested and add the sessionNavigationBar template

File: templates/index.html (hunk 2 above)
Comment (DelaMarc) [reply templates/index.html:18]: I pushed the branch

File: server.py (hunk 1 above)
Comment (DelaMarc) [reply server.py]: For nothing. My bad, it's removed now.

**Review by YashasG98 — APPROVED** [review]

**Pull request merged**